### PR TITLE
refactor(rxAciveUrl): remove rxActiveUrl from components

### PR DESCRIPTION
**Pull request by kevincareer427**
JIRA: https://jira.rax.io/browse/FRMW-210

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM

### CODE REVIEW

**Comment by kevincareer427** [comment]
migration guide has been updated: 
https://github.com/rackerlabs/encore-ui/wiki/2.0-Migration-Guide

**Comment by Droogans** [comment]
I like the look of this.

**Comment by CITguy** [comment]
LGTM

File: src/components/rxActiveUrl/docs/rxActiveUrl.midway.js [hunk 1]
```diff
@@ -1 +0,0 @@
-// Midway tests for the example/demo component page go here
```
Comment (Droogans) [inline src/components/rxActiveUrl/docs/rxActiveUrl.midway.js]: I'm a little hesitant to drop this file here.

**Comment by parlarjb** [comment]
LGTM (I'm marking the Vidyo checkbox, I don't think this needs a vidyo walk through, unless you'd like to do one)

**Comment by glynnis** [comment]
LGTM. Probably didn't need a Design LGTM on this one, FWIW.

**Pull request merged**